nautobot_design_builder/tests/test_builder.py

```diff
@@ -199,13 +199,13 @@
 
 INPUT_ASSIGN_TAGS = """
 tags:
-  - name: Test Tag
+  - "!create_or_update:name": Test Tag
```

Comment:
to comment with @abates , without an "identifier", I can't compare new and previous designs. Thus, every design item should come with an identifier.